packages/react-components/react-progress/bundle-size/Progress.fixture.js

```diff
@@ -0,0 +1,7 @@
+import { Progress } from '@fluentui/react-progress';
```

Comment:
Please also add `bundle-size` script to `package.json` so it will run on CI:

https://github.com/tomi-msft/fluentui/blob/30b94ecfd7386c6cb0728d45a4ec1ef503bd76f9/packages/react-components/react-input/package.json#L16